tests/test_on_data/test_format_converters/spikeglx_multi_probe_metadata.json

```diff
@@ -0,0 +1,63 @@
+{
+  "NWBFile": {
+    "session_description": ""
+  },
+  "Ecephys": {
+    "Device": [
+      {
+        "name": "NeuropixelImec0",
+        "description": "{\"probe_type\": \"0\", \"probe_type_description\": \"NP1.0\", \"flex_part_number\": \"NP2_FLEX_0\", \"connected_base_station_part_number\": \"NP2_QBSC_00\"}",
```

Comment:
Question: why is this adding the escape character everywhere?

Reply:
B/c it's a double quote in another quote?

I don't think about it too hard, just copied/pasted content from `json.dumps(metadata, indent=2)`

Maybe `json.dump` (no `s`) would look different but parse the same?

Reply:
All right. I will check on it on a future refactor.